app/uk/gov/hmrc/cataloguefrontend/DependencyExplorerController.scala

```diff
@@ -100,10 +100,21 @@ class DependencyExplorerController @Inject() (
         scopes         =  DependencyScope.values
         repoTypes      =  RepoType.values
         groupArtefacts <- dependenciesService.getGroupArtefacts
+        filledForm     = {
+          SearchForm(
+            group         = group,
+            artefact      = artefact,
+            versionRange  = versionRange.getOrElse(BobbyVersionRange("[0.0.0,]").range),
+            team          = team.getOrElse(""),
+            flag          = flag.getOrElse(SlugInfoFlag.Latest.asString),
+            scope         = `scope[]`.getOrElse(List(DependencyScope.Compile.asString)),
+            repoType      = `repoType[]`.getOrElse(List(RepoType.Service.asString))
+          )
+        }
         res <- {
           def pageWithError(msg: String) =
             page(
-              form().bindFromRequest().withGlobalError(msg),
+              form().withGlobalError(msg),
```

Comment:
Is this missing `bindFormRequest` (or `.fill(filledForm)`) - otherwise it's reporting an error and not showing what the user submitted to be corrected?